lib/paper_trail/events/destroy.rb

```diff
@@ -14,7 +14,7 @@ class Destroy < Base
       def data
         data = {
           item_id: @record.id,
-          item_type: @record.class.base_class.name,
+          item_type: @record.class.base_class.polymorphic_name,
```

Comment:
Should this be just `@record.class.polymorphic_name`? The [default implementation of `polymorphic_name` already considers the `base_class`](https://github.com/rails/rails/blob/e9f38a164e36104cc495d5ac5859e57f6be75f36/activerecord/lib/active_record/inheritance.rb#L210), and if you have overridden it you may have a reason to do so only in the child class.